docs/src/main/asciidoc/attributes.adoc

```diff
@@ -5,3 +5,8 @@
 :idseparator: -
 :icons: font
 :imagesdir: ./images
+
+// Product-related attributes
+:ProductName: Quarkus
+:ProductLongName: Red Hat build of Quarkus
```

Comment:
In the community, I think this will just be Quarkus. We will have a separate attributes files downstream.

Reply:
Ah, thanks:) I'll change the value of `ProductLongName` to match the value of `Quarkus`. this was originally to test the conditionals in the document, but I should have removed it. No need to have the product name in the community repo, after all

Reply:
I'll change it to say `Quarkus`. no need to have the product name in the community repo, after all. thank you.

Reply:
> > @ncbaratta Thank you for the review. I have incorporated your suggestions and added additional correction that align with the IBM style guide.
> 
> I see a few of my comments are still unaddressed here: https://github.com/quarkusio/quarkus/pull/14306/files



> > @ncbaratta Thank you for the review. I have incorporated your suggestions and added additional correction that align with the IBM style guide.
> 
> I see a few of my comments are still unaddressed here: https://github.com/quarkusio/quarkus/pull/14306/files

Oh, thanks. I must have missed those. I'll address them in a follow-up commit.

Reply:
Thanks:) removed it:)